docs/area-owners.md

```diff
@@ -142,10 +142,10 @@ Note: Editing this file doesn't update the mapping used by `@dotnet-policy-servi
-| area-TypeSystem-coreclr                        | @steveisok           | @davidwrighton @MichalStrehovsky @janvorli @mangod9 @dotnet/area-type-system-and-startup |                                                                                                                                                                                                                                                                                           |
-| area-UWP                                       | @tommcdon            | @dotnet/area-uwp                                                                         | UWP-specific issues including Microsoft.NETCore.UniversalWindowsPlatform and Microsoft.Net.UWPCoreRuntimeSdk                                                                                                                                                                              |
+| area-Tracing-coreclr                           | @steveisok            | @dotnet/area-tracing-coreclr                                                             | .NET runtime issues for EventPipe and ICorProfiler                                                                                                                                                                                                                                        |
+| area-Tracing-mono                              | @steveisok            | @dotnet/dotnet-diag @thaystg                                                                       |                                                                                                                                                                                                                                                                                           |
+| area-TypeSystem-coreclr                        | @mangod9           | @davidwrighton @MichalStrehovsky @janvorli @mangod9 |                                                                                                                                                                                                                                                                                           |
+| area-UWP                                       | @steveisok            | @dotnet/area-uwp                                                                         | UWP-specific issues including Microsoft.NETCore.UniversalWindowsPlatform and Microsoft.Net.UWPCoreRuntimeSdk                                                                                                                                                                              |
```

Comment:
```suggestion
```
I do not think we need UWP area. There are no issues under in.

Reply:
Actually, there are a few old issues - I have marked them as backlog cleanup candidate.